dataspammer.bat

```diff
@@ -69,41 +69,33 @@
         set "sudo=0"
     )
 
-    :sudo.parse
-    :: sudo requires a Wrapper to work in Powershell (sudo.ps1) maybe sudo will work without it natively in Batch
-    :: else i will need to convert sudo.ps1 to Batch
+:sudo.parse
+    :: sudo features a Powershell Wrapper (sudo.ps1) to make it more User-Friendly.
+    :: I Hope its not neccessarily needed for sudo to work, because converting it to Batch is hard. (base64 conding)
```

Comment:
Converting to batch impossible without embedded .dll / .exe is impossible (future releases use pwsh also maybe fix?)